saleor/graphql/schema.graphql

```diff
@@ -15963,7 +15963,6 @@ type WebhookDryRunError {
 """An enumeration."""
 enum WebhookDryRunErrorCode {
   GRAPHQL_ERROR
-  UNABLE_TO_PARSE
```

Comment:
WIll be now under `GRAPHQL_ERROR`, but the message will point accurately to error reason

Reply:
Actually, it's not a bad idea to return a more specific error, message is only for dev, while the code is for the frontend to know which error to display. The more accurate code, the easier it is for them to handle the error.